crypto/crypto/src/traits.rs

```diff
@@ -28,20 +28,24 @@ pub enum CryptoMaterialError {
     DeserializationError,
     /// Key or signature material deserializes, but is otherwise not valid.
     ValidationError,
-    /// Key or signature material does not have the expected size.
+    /// Key, threshold or signature material does not have the expected size.
     WrongLengthError,
     /// Part of the signature or key is not canonical resulting to malleability issues.
     CanonicalRepresentationError,
     /// A curve point (i.e., a public key) lies on a small group.
     SmallSubgroupError,
     /// A curve point (i.e., a public key) does not satisfy the curve equation.
     PointNotOnCurveError,
+    /// BitVec mismatch error in accountable multi-sig schemes.
```

Comment:
Can this be more specific? Unless I misunderstand, the only thing that can go wrong is if there's a nonzero bit in a position >= keys.length. If that's correct, maybe we can call this something like OutOfBoundsBitVectorIndex?